pkg/engine/workflow/engine_test.go

```diff
@@ -175,7 +175,7 @@ func TestExecutePlan(t *testing.T) {
 			expectedStatus: &v1beta1.PlanStatus{
 				Status: v1beta1.ExecutionInProgress,
 				Name:   "test",
-				Phases: []v1beta1.PhaseStatus{{Name: "phase", Status: v1beta1.ExecutionInProgress, Steps: []v1beta1.StepStatus{{Status: v1beta1.ErrorStatus, Name: "step"}}}},
+				Phases: []v1beta1.PhaseStatus{{Name: "phase", Status: v1beta1.ExecutionInProgress, Steps: []v1beta1.StepStatus{{Status: v1beta1.ErrorStatus, Name: "step", Message: "A transient error when executing task test.phase.step.task. Will retry. dummy error"}}}},
```

Comment:
Let's not call it a dummy error - it sounds more like we threw it just to throw it, so ignore it, when it's a true error, just transient (as you mention). "Retrying until this step succeeds" or something.

Reply:
It's only in the test 😉

Reply:
yeah I am keeping it since it's just tests so I think it does not matter :) let me know if there's really a strong reason to change this